demo_scripts/backtester/compare_different_initial_risks.py

```diff
@@ -18,7 +18,9 @@
 from demo_scripts.backtester.moving_average_alpha_model import MovingAverageAlphaModel
 from demo_scripts.common.utils.dummy_ticker import DummyTicker
 from demo_scripts.demo_configuration.demo_data_provider import daily_data_provider
-from demo_scripts.demo_configuration.demo_ioc import container
+from demo_scripts.demo_configuration.demo_settings import get_demo_settings
+from documents_utils.document_exporting.pdf_exporter import PDFExporter
+from documents_utils.excel.excel_exporter import ExcelExporter
```

Comment:
This import is not working for me,
I think you have to add `from qf_lib.document_utils.document_exporting.pdf_exporter import PDFExporter` which is used in the rest of the repository